docs/DeveloperGuide.md

```diff
@@ -495,76 +507,114 @@ The sequence diagram below shows the interactions between objects during the exe
 ![ExitSequenceDiagram](images/ExitFeatureSequenceDiagram.jpg)
 
 
-### \[Proposed\] Undo/redo feature
+### Undo/Redo features
+
+In this section, the functionality of the undo and redo features, the expected execution path, the structure of the UndoCommand and RedoCommand class, and the interactions between objects with the UndoCommand and RedoCommand objects will be discussed.
+
+#### What are the undo and redo features
+
+The undo feature allows users reverse an `add`, `edit`, or `delete` command.
+
+The redo feature allows users to reverse an `undo` command. 
+
+#### Execution of undo feature
+
+The `undo` command is as follows:
+
+`undo`
+
+Calling this command undoes only the last executed add, edit or delete command, and can only be used after executing an add, edit, or delete command.
+
+This command cannot be used in succession to undo previously executed commands besides the last executed one.
```

Comment:
Good that you alert the reader. If you would like, you can use icons to draw reader's attention.